<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login - SST</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3383ff;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>110</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>110</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>700</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #f9f9f9;
border-radius: 15px;
border: 1px solid #cccccc;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,2,2,1,2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Carlito</family>
           <pointsize>72</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #3383ff;
border: none;</string>
         </property>
         <property name="text">
          <string>SST</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Carlito</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #3383ff;
border: none;</string>
           </property>
           <property name="text">
            <string>USU√ÅRIO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="userLoginLineEdit">
           <property name="font">
            <font>
             <family>Carlito</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #3383ff;
padding: 10px;
border-radius: 5px;
border: 1px solid #3383ff;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Carlito</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #3383ff;
border: none;</string>
           </property>
           <property name="text">
            <string>SENHA</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLineEdit" name="userPassLineEdit">
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #3383ff;
padding: 10px;
border-radius: 5px;
border: 1px solid #3383ff;</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::EchoMode::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="HoverWidget" name="showPassBtn">
             <property name="styleSheet">
              <string notr="true">background: grey;
border-radius: 2px;
padding: 0 10px;</string>
             </property>
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="HoverWidget" name="loginBtn">
           <property name="styleSheet">
            <string notr="true">background: grey;
border-radius: 5px;
padding: 0 10px;</string>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HoverWidget</class>
   <extends>QPushButton</extends>
   <header>hoverwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
